proxy/server/server.go

```diff
@@ -7,6 +7,10 @@ import (
 	"net/http"
 	"sync"
 
```

Comment:
In a couple of files I noticed that stdlib section is split in two. We need to remove that empty line and re run `goimport` to sort them properly.